New package: OCBase.OCCT.Personal version 12.1.18

Dragon1573 opened the pull request:
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] This resolves #147718

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

Dragon1573 committed:
New package: OCBase.OCCT.Personal version 12.1.18

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

Trenly commented on manifests/o/OCBase/OCCT/Personal/12.1.18/OCBase.OCCT.Personal.locale.en-US.yaml:
```diff
@@ -0,0 +1,17 @@
+LicenseUrl: https://www.ocbase.com/purchase
+Copyright: Copyright ©® OCBASE - 2023 until the end of time
+ShortDescription: OCCT is the most popular all-in-one stability check & stress test tool available.
+ReleaseNotesUrl: https://www.ocbase.com/download
```
This doesn't lead to the release notes

Trenly commented on manifests/o/OCBase/OCCT/Personal/12.1.18/OCBase.OCCT.Personal.locale.en-US.yaml:
```diff
@@ -0,0 +1,17 @@
+PackageName: OCCT
+PackageUrl: https://www.ocbase.com/download
+License: Proprietary
+LicenseUrl: https://www.ocbase.com/purchase
```
This should be the PurchaseUrl, not the LicenseUrl

Dragon1573 replied:
At the end of this page, there's an EULA button. EULA is provided as an embedded panel in that page instead of a standalone page. So I place it as license URL. :disappointed:

Dragon1573 committed:
Accept suggestions from Change Requests

- Remove `ReleaseNotes` from locale YAML
- Change `LicenseUrl` to `PurchaseUrl`
